pootle/apps/pootle_translationproject/views.py

```diff
@@ -128,18 +128,14 @@ def browse(request, translation_project, dir_path, filename=None):
     ctx.update(get_browser_context(request))
 
     # TODO improve plugin logic
-    if "import_export" in settings.INSTALLED_APPS and request.user.is_authenticated():
-        from import_export.views import handle_upload_form
-
-        ctx.update(handle_upload_form(request, project))
-
-        has_download = (not translation_project.is_terminology_project and
-                        (check_permission('translate', request) or
-                         check_permission('suggest', request)))
-        ctx.update({
-            'display_download': has_download,
-            'has_sidebar': True,
-        })
+    if "import_export" in settings.INSTALLED_APPS:
+        if not translation_project.is_terminology_project:
```

Comment:
IMHO we must allow people to download terminology. I am not 100% sure we should forbid people to upload it.